gossip/src/crds_value.rs

```diff
@@ -169,6 +171,9 @@ pub struct SnapshotHash {
 impl Sanitize for SnapshotHash {
     fn sanitize(&self) -> Result<(), SanitizeError> {
         sanitize_wallclock(self.wallclock)?;
+        if self.hashes.len() > MAX_SNAPSHOT_HASHES {
+            return Err(SanitizeError::ValueOutOfBounds);
+        }
```

Comment:
`MAX_SNAPSHOT_HASHES` is _supposed to be_ the largest number of hashes that you can include in SnapshotHash and send it through push messages or pull responses while the whole thing fits into a packet:
https://github.com/solana-labs/solana/blob/d6fdc4b63/gossip/src/cluster_info.rs#L3170-L3194

But if you have already received a packet and successfully deserialized it, then there is no point in checking the number of hashes in the sanitize here.
Regardless, this change is not related to this commit, so please remove it.

Reply:
Done!